pkg/api/schema/schema.proto

```diff
@@ -1439,6 +1454,19 @@ message RetryInfo {
   int32 retry_delay = 1;
 }
 
+message TruncateDatabaseRequest {
+  // Database name
+  string database = 1;
+
+  // Retention Period of data
+  NullableMilliseconds retentionPeriod = 2;
```

Comment:
if the truncator process is not enabled and retentionPeriod is not specified in the request, which value is used? or it returns an error?

Reply:
This is used in the Truncate command in the immuadmin. If retention period is not specified, an error is returned